### ISSUE TITLE
[EPIC] Parallel Replication

### ISSUE THREAD

**Issue by cgardens** [airbytehq/airbyte#7750]
## Tell us about the problem you're trying to solve
This issue is an index to various projects around parallel data replication in Airbyte. This is a broad topic and there are several approaches that we can take to target certain issues. 

Parallelization is focused on increasing performance of replication. There are 3 avenues to think along:
1. handle a source with alot of streams, but each individual stream is a manageable size (i.e. we want to replicate multiple streams in parallel)
2. handle a source that has one huge stream (i.e. we want to replicate that stream's data in parallel)

issues:
- https://github.com/airbytehq/airbyte/issues/4081
- https://github.com/airbytehq/airbyte/issues/4081

**Comment by tonyxiao** [comment]
What's the 3rd avenue to think along? Or is this a classic off-by-one? ;)

**Comment by jordandakota** [comment]
> What's the 3rd avenue to think along? Or is this a classic off-by-one? ;) 

Maybe it's A, B, A+B

**Comment by Udit107710** [comment]
@cgardens Any progress on this epic?

**Comment by cgardens** [comment]
Yes, a couple sources (e.g. stripe) already support this and we are migrating others over to it. It's using our concurrent cdk.

**Comment by davinchia** [comment]
This is out of date with the new speed project. Closing.
